cmd/rotate_certs.go

```diff
@@ -389,11 +403,11 @@ func (rcc *rotateCertsCmd) rotateEtcd(ctx context.Context) error {
 		}
 	}
 
-	log.Infoln("Rebooting all nodes... This might take a few minutes")
-	err := rcc.rebootAllNodes(ctx)
```

Comment:
Must the nodes be rebooted? Could we just restart some services?

Reply:
They don't have to be rebooted, that is why i left that part of the code out. However i will be interested to know if anyone has any reason why it should be

Reply:
@jackfrancis or @mboersma do you folks know why nodes needed to be rebooted?

Reply:
nope